voxel51/query.py

```diff
@@ -172,8 +172,8 @@ class AnalyticsQuery(BaseQuery):
     Attributes:
         fields (list): the list of fields to include in the returned records.
             The supported query fields are `id`, `name`, `version`,
-            `upload_date`, `description`, `scope_name`, `supports_cpu`,
-            `supports_gpu`
+            `upload_date`, `description`, `scope`, `supports_cpu`,
```

Comment:
forgot to add `pending` to the docstring. No broken windows! https://blog.codinghorror.com/the-broken-window-theory/.

Reply:
also, shouldn't `scope_name` have been updated to `scope`?